config.go

```diff
@@ -28,6 +28,8 @@ type Config struct {
 	PrintFailedTestsOnly bool
 	TestDirectory        string
 	Verbosity            int
+	EnableRetries        bool
```

Comment:
Maybe I am missing something but where are we counting the number of tries? I see that we have a max, but I don't see where we are checking to see if we hit that max.

Reply:
Retries are handled by the library https://github.com/hashicorp/go-retryablehttp. That get's setup here https://github.com/nytimes/httptest/blob/c77093cfc1045e5ce29b0f970e96d179cd20ad93/http.go#L111-L118. It uses the [retry callback](https://github.com/nytimes/httptest/blob/c77093cfc1045e5ce29b0f970e96d179cd20ad93/tester.go#L64-L78) to determine if the tests failed. If it fails it will retry up the the max number of retries.